docs/registries.conf.5.md

```diff
@@ -0,0 +1,28 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is YAML.
+
+# FORMAT
+The [YAML_format][yaml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`.
+
+The following example configuration defines two registries and one insecure registries.
+
+```
+registries:
+    - registry1.com
+    - registry2.com
+
+insecure_registries:
+    - registry3.com
+```
```

Comment:
What I'm concerned about is not the now, but the later.  A year from now someone finds your blog via googling, is trying out CRI-O for the first time and the registries stuff doesn't work.  Bad taste in the mouth.  If you still want to get your blog out ASAP, then I think we should at least mention the coming change.  FWIW that's my $.02.